.github/actions/build/action.yaml

```diff
@@ -146,15 +150,15 @@ runs:
             BUILD_PLATFORM="android"
           else
             BUILD_PLATFORM="linux"
-            cd src/
+            cd cobalt/src/
```

Comment:
nit: I would recommend to use `pushd cobalt/src` and then popd instead of `cd ../../`